Doc/library/secrets.rst

```diff
@@ -52,7 +52,7 @@ randomness that your operating system provides.
 
 .. function:: randbits(k)
 
-   Return an int with *k* random bits.
+   Returns a non-negative Python integer with *k* random bits.
```

Comment:
```suggestion
   Returns a non-negative int with *k* random bits.
```

Reply:
Should we amend the docs there as well then: [docs.python.org/3/library/random.html#random.getrandbits](https://docs.python.org/3/library/random.html#random.getrandbits)?